src/dictionaries.py

```diff
@@ -34,11 +34,19 @@
 ]
 
 # Add a new waypoint to the list
-# YOUR CODE HERE
+waypoints.append({
+    "lat": 23,
+    "lon": 33,
+    "name": "a fourth place"
+})
 
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
-# YOUR CODE HERE
+for waypoint in waypoints:
+    if waypoint.get('name') == 'a place':
+        waypoint["name"] = 'not a real place'
+        waypoint["lon"] = -130
 
 # Write a loop that prints out all the field values for all the waypoints
-# YOUR CODE HERE
\ No newline at end of file
+for item in waypoints:
+    print(item.values())
```

Comment:
item.items() would print out all of the keys and values here.

Reply:
You're right. It probably would make more sense to have the key and values showing, but it mentioned only values, so I may have taken it a little too literally.